### ISSUE TITLE
More than one remote source

### ISSUE THREAD

**Issue by pawelkasperek** [gmasterscm/ama#13]
Hi,

Sometimes I use more than one remote source ex. origin and upsteam have differences URL. I'm added these in git ref but I cannot change in gmaster. Is possible to choose remote reference source ?

**Comment by psantosl** [comment]
Right, we will add some remote management quite soon. In fact we already have some ideas (and initial implementation) of how to do it.
